ecma48/parser.go

```diff
@@ -257,11 +260,18 @@ func (p *Parser) stateEscape(r rune) {
 	case 0x30 <= r && r <= 0x4F || 0x51 <= r && r <= 0x57:
 		// TODO: p.dispatchEsc()
 		p.state = stateGround
+	case r == '(':
+		p.state = stateCharSet
 	default:
 		log.Printf("? ESC %q", r)
 	}
 }
 
+func (p *Parser) stateCharSet(r rune) {
+	p.state = stateGround
+	log.Printf("? (char set) ESC ( %q", r)
+}
+
```

Comment:
Not sure if this should be named to `stateCharSetG0` since there is a `ESC %` that starts the sequence selecting character set (https://man7.org/linux/man-pages/man4/console_codes.4.html)

Reply:
Good catch! I'll fix this

Reply:
Fixed!